extensions/replication/queueProcessor/QueueProcessor.js

```diff
@@ -15,6 +15,67 @@ const BackbeatConsumer = require('../../../lib/BackbeatConsumer');
 const BackbeatClient = require('../../../lib/clients/BackbeatClient');
 const QueueEntry = require('../utils/QueueEntry');
 
+class _UserAuthManager {
+    constructor(authConfig, log) {
+        assert.strictEqual(authConfig.type, 'user');
+
+        this.log = log;
+        const userInfo = authdata.users.find(
+            user => user.name === authConfig.user);
+        if (userInfo === undefined) {
+            throw Error(`No such user registered: ${authConfig.user}`);
```

Comment:
Ok, since this impacts only open-source users and it would happen only when there is misconfiguration, let's keep it and cleanup later.